src/app/TideSearchApplication.cpp

```diff
@@ -1930,8 +1929,7 @@ void TideSearchApplication::calcResidueScoreCount (
         if (newCol == colLast) {
```

Comment:
One thing that I think you missed that I didn't make clear in the last comment. I also suggested switching the order between the if and else if statement.

I believe "if (col <colLast)" would be faster than "if (coll==colLast)" as we expect the first one to be true more often.